extensions/amp-story/0.1/amp-story.js

```diff
@@ -133,6 +133,9 @@ export class AmpStory extends AMP.BaseElement {
 
     /** @private {?AmpStoryPage} */
     this.activePage_ = null;
+
+    this.background_ = new AmpStoryBackground(this.element);
```

Comment:
Is there a way we can avoid managing this in the non-desktop case, to save resources?